aspnetcore/includes/blazor-preview-notice.md

```diff
@@ -0,0 +1,6 @@
+> [!WARNING]
+> **Blazor is an unsupported experimental web framework. Blazor is in preview.**
+>
+> Blazor preview documentation is provided to assist you in trying out Blazor. Blazor isn't a committed Microsoft product and isn't supported by a Microsoft SLA at this time. Blazor's feature set, capabilities, and tooling are subject to change at any time without notice. Blazor shouldn't be used for production workloads. Microsoft makes no warranties, express or implied, with respect to the information provided in this preview documentation or for the Blazor technology itself.
+>
+> We welcome feedback on the docs. Open an issue in the [aspnet/Docs repo](https://github.com/aspnet/Docs/issues/new). To report bugs and make suggestions for Blazor, open an issue in the [aspnet/Blazor repo](https://github.com/aspnet/Blazor/issues/new).
```

Comment:
I don't think we need to invite doc feedback here specifically. We invite feedback on our docs in general, not just for Blazor. Inviting folks to open issues in the Blazor repo is fine.